client/src/components/DatasetInformation/DatasetInformation.vue

```diff
@@ -37,13 +37,25 @@
                     <tr v-if="dataset.id">
                         <td>History Content API ID</td>
                         <td>
-                            <div id="dataset-id">{{ dataset.id }} <decoded-id :id="dataset.id" /></div>
+                            <div id="dataset-id">
+                                {{ dataset.id }} <decoded-id :id="dataset.id" /> (<a
```

Comment:
This should go into the JobInformation.vue file.

Reply:
addressed